src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -361,16 +361,17 @@ internal FunctionLoadRequest GetFunctionLoadRequest(FunctionMetadata metadata, M
         internal void LoadResponse(FunctionLoadResponse loadResponse)
         {
             _functionLoadRequestResponseEvent?.Dispose();
-            _workerChannelLogger.LogDebug("Received FunctionLoadResponse for functionId:{functionId}", loadResponse.FunctionId);
+            string functionName = _functions.Single(m => m.GetFunctionId().Equals(loadResponse.FunctionId, StringComparison.OrdinalIgnoreCase)).Name;
+            _workerChannelLogger.LogDebug("Received FunctionLoadResponse for function: {functionName} with functionId:{functionId}", functionName, loadResponse.FunctionId);
```

Comment:
nit: can you put a space after the second `:` -- and use `function id` like you did below? I think it'd make it a tiny bit easier to parse logs for this.

Also -- can you put single quotes around all of the identifiers? Same thing -- using parse in kusto can be easier that way if you know it's surrounded by single quotes (as opposed to ending with a space or a `.`).

Reply:
done!